docs/quick-start/project-structure.md

```diff
@@ -0,0 +1,197 @@
+# Project Structure
+
+When creating a new React Native project with Ignite, your project will include a number of nested files and folders. This structure is one the React Native best practices included with Ignite. With this structure, your project already has a head start. Let's take a closer look at Ignite's structure and explain how to leverage its benefits.
+
+## Ignite React Native vs Vanilla React Native
+
+First let's take a look at a "Vanilla" React Native project's structure. To start, navigate to directory you'd like to create a new React Native project and run the `react-native init` command to create a project without Ignite.
+
+```
+cd projects
+react-native init VanillaProject
+cd VanillaProject
+```
+
+Your new React Native project should have a file structure similar to the following:
+
+```
+VanillaProject
+├── __tests__
```

Comment:
In the `unholy-app-template`, we use `ava` for testing.   What you see here is Jest's folder.  You get jest when you pick `minimal` or `empty`.

We're going to evaluate whether to switch up `ava` for `jest` now that React Native ships with it.

That will be post 2.0 launch.